### ISSUE TITLE
how to integrate in swift 3

### ISSUE THREAD

**Issue by SagarDabhi** [WirecardMobileServices/SuprAcceptSDK-iOS#1]
Hello Team,

Is there any example in swift 3 because i am not aware how to integrate in swift.

**Comment by WirecardMobileServices** [comment]
Hi, did you have a look at the integration documentation https://wirecardmobileservices.github.io/SuprAcceptSDK-Developer/

There are gists for Objective-C as well as for Swift

**Comment by SagarDabhi** [comment]
Are you talkig about "pod 'AcceptSDK2/All" pod ? , currently i am trying with "pod "SUPRAcceptSDK" this

**Comment by WirecardMobileServices** [comment]
Apologies, the doco will be updated to pod 'SUPRAcceptSDK/All'

**Comment by SagarDabhi** [comment]
Thanks for that, but i am looking for demo so that i can understand how to integrate because i am totally not aware integration of wirecard.

**Comment by WirecardMobileServices** [comment]
The SDK does not have the Demo Payment application yet. 
You can check the Example folder - run pod install in there and use the resulting workspace to see how each SDK API is used. Check the TestDemo target and all available tests - these show how to use each API.

The integration documentation can take you through the setup, initialization and use of each API you wish to use.

I would start with the integration document and go through each section.

Alternatively we will be offering our Whitelabel solution by the end of November  - with that we will be able to brand our solution to your needs.

**Comment by SagarDabhi** [comment]
Thanks @WirecardMobileServices , one more question "is this available for iOS 11 and swift 4?".

**Comment by RadoslavDanko** [comment]
Hi, we are using our SDK in our Whitelabel solution and running on iOS 11 with no issues.
As for Swift 4 - you are able to use the SDK the same way as you would for Swift 3 :-)
You can have a look at the gists written in Swift:
https://wirecardmobileservices.github.io/SuprAcceptSDK-Developer/int-salerequest-cash/
and switch the platform selector to iOS (default) and language selector to Swift.

Any issues, just let me know.

**Comment by SagarDabhi** [comment]
Thank you very much @RadoslavDanko , i will try.

**Comment by SagarDabhi** [comment]
Hello @RadoslavDanko  & @WirecardMobileServices  i am Facing issue after installation pod : 
 ==>Demos/paymentDemo/Pods/SUPRAcceptSDK/Pod/Classes/frameworks/Accept.framework/Accept compiled with newer version of Swift language (unknown ABI version 0x05) than previous files (3.0) for architecture x86_64
clang: error: **linker command failed with exit code 1 (use -v to see invocation)**

**Comment by RadoslavDanko** [comment]
Check if your project has the Swift language version setting 3.2
If it does then clean your derived data and try to build again please .

**Comment by SagarDabhi** [comment]
No i am using swift 3.0. and already done all the process derived data & project clean.

**Comment by RadoslavDanko** [comment]
Can you update your project Swift language version to 3.2 then please ?

**Comment by SagarDabhi** [comment]
is there any other solution for that?

**Comment by RadoslavDanko** [comment]
Think not, as SDK was compiled for Swift 3.2 (at that time the latest Swift available) one would need to use XCode 9, set the Swifth language to 3.2 and convert the syntax of your app from 3.0 to 3.2 - there should not be that many changes

**Comment by SagarDabhi** [comment]
@RadoslavDanko  Thanks for reply, will try asap.

**Comment by optimumbrew** [comment]
I tried to implement quickblox - video chat function into my ios app with Swift3. After test, I could call the opponent and the opponent could accept the call, but there is not video and audio show up. I confirmed the functions "receivedRemoteVideoTrack" and "receivedRemoteAudioTrack" are executed, but still no audio and video in the call. Could anyone advise? Thank you very much!

**my output are attach**
![image1](https://user-images.githubusercontent.com/34236449/33831923-ffee9460-de9f-11e7-884b-7570700d1903.PNG)
